src/gui/optionsdialog.ui

```diff
@@ -2156,6 +2199,9 @@
               <layout class="QVBoxLayout" name="verticalLayout_14">
                <item>
                 <widget class="QCheckBox" name="checkDHT">
+                 <property name="toolTip">
+                  <string>Find peers seperate from tracker</string>
```

Comment:
maybe: "Find peers on the DHT network"?
and typo: "seperate" -> "separate"

Reply:
Sounds good to me. Done.